share/revng/pipelines/revng-pipelines.yml

```diff
@@ -660,6 +660,65 @@ Branches:
           SingleTargetFilename: mlir-llvm-dialect-with-clift-types.mlir
           Docs: |-
             A test artifact to import types into a Clift module.
+    # New clift-based decompilation pipeline
```

Comment:
I'd settle for "clift-based" vs "legacy" decompilation pipeline.
Let's avoid new, otherwise it will stick around forever.